util/build_configs/module-functions.bash

```diff
@@ -0,0 +1,178 @@
+
+# Source this to get bash functions for manipulating Cray system modules with error checking
```

Comment:
I wrote a python module for very similar functionality back in my BTR days: https://github.com/ben-albrecht/environmentmodules 

Not feedback, just an interesting tidbit / shameless plug.